src/templates/kit/screens/ReviewScreen.js

```diff
@@ -18,8 +23,35 @@ function Review({
     next(data)
   }, [data, next])
 
+  const containerRef = useRef()
+  const prevNextRef = useRef()
+
+  useEffect(() => {
+    if (containerRef.current) {
+      containerRef.current.focus()
+    }
+  }, [])
```

Comment:
Should this effect have `containerRef` as dependency?

Reply:
Probably not necessary :+1:

Reply:
Why is it not necessary? Refs don't start out with `current => null` and update afterwards? In that case it would be necessary to have it as a dependency no?